integration-tests/api/__tests__/admin/product.js

```diff
@@ -56,7 +56,7 @@ medusaIntegrationTestRunner({
       await createAdminUser(dbConnection, adminHeaders, container)
     })
 
-    describe("/admin/products", () => {
+    describe.skip("/admin/products", () => {
```

Comment:
I wanted to have a relatively small PR so we can merge progressively.

Since it's WIP there are few tests that are failing due to me commenting out some test code. Once both APIs are compatible I'll remove this. I think I'll be able to do this tomorrow, so if we're ok with not having these tests running for a couple of days we can leave it as it is, or I can just wait to merge the PR once all of the changes are in place.